extensions/lifecycle/tasks/LifecycleUpdateTransitionTask.js

```diff
@@ -112,7 +112,7 @@ class LifecycleUpdateTransitionTask extends BackbeatTask {
     }
 
     _garbageCollectLocation(entry, locations, log, done) {
-        const { bucket, key, version, eTag } = entry.getAttribute('target');
+        const { bucket, key, version, eTag, accountId } = entry.getAttribute('target');
```

Comment:
`target.owner` is needed to get the [backbeat client](https://github.com/scality/backbeat/pull/2191/files#diff-7beaf0ffcd262dad5a78c1b25cb7d54b9ac577441b372167f96d8c182be8fcf7R23) for the Garbage Collector task.